Core/Core/AppEnvironment/AppEnvironment.swift

```diff
@@ -72,6 +72,7 @@ open class AppEnvironment {
         OfflineModeAssembly.make()
         api = API(session)
         currentSession = session
+        currentSession?.migrateSavedAnnotatedPDFs()
```

Comment:
I chose to call this here, and do it per session so to be targeting those folders relevant to this specific feature & session. Thus we do it lazily if there are multiple sessions.
There was an attempt to do it on app launch for all sessions at once, but there was a difficulty fetching all sessions that have files saved to `Documents` directory. Not all of them get persisted to Keychain as I was expecting.